### ISSUE TITLE
Minor issue with install instructions

### ISSUE THREAD

**Issue by amouat** [windmilleng/tilt.build#280]
The following step didn't work for me:

```
sudo snap install microk8s --classic && \
microk8s.enable dns && \
microk8s.enable registry
```
The problem is that my user wasn't in the microk8s group, so the enable commands don't work. You could change it to:

```
sudo snap install microk8s --classic && \
sudo microk8s.enable dns && \
sudo microk8s.enable registry
```
But I'm not sure that's the best solution.

**Comment by nicks** [comment]
Thanks for the report! Ya, you're right, will update that

**Comment by nicks** [comment]
fixed in #281
